src/main/java/org/eclipse/microprofile/starter/core/model/MicroProfileVersion.java

```diff
@@ -30,7 +32,7 @@
     // Order is important as it determines the order in the Dropdown on the screen.
     // @formatter:off
     NONE(null, "")
-    , MP30("3.0", "MP 3.0")
+    , MP30("3.0", "MP 3.0", Collections.singletonList("mp3_0"))
```

Comment:
The new Health template should apply to MP Health 2.0 and above. It might be better to use mpHealthReady2_0 and mpHealthLive2_0 for two templates to be used. Can you also rename the newly added ServiceHealthCheck to ServiceLiveHealth and add a new file named ServiceReadyHealth. Also in the body, can you also add .data("ready", true) or .data("live", true) accordingly?

Reply:
We only select umbrella versions, so it is logical that templates are grouped by this umbrella version.

As discussed with Karm, I'm going to name it "mp3_x" so that it isn't only tied to MP 3.0.